docs/reference/cli-reference/cli-ref-sources.md

```diff
@@ -69,12 +69,16 @@ where `<operation>` is one of *List, Add, Remove, Enable, Disable,* or *Update*,
 
   Comma-separated list of valid authentication types for this source. By default, all authentication types are valid. Example: `basic,negotiate`.
 
+- **`-ProtocolVersion`**
+
+  The NuGet server protocol version to be used. The current version is "3". Supported in NuGet 3.0+. Defaults to "2" if not specified.
```

Comment:
The "Supported in NuGet 3.0+" could be interpreated as saying that protocol version 3 is supported in NuGet 3.0+, or that the `-ProtocolVersion` option is supported in NuGet 3.0+.  However, `-ProtocolVersion` won't be supported until NuGet 6.5 or 6.6, depending when the PR gets merged.

Given that this is a brand new option, and many customers reading the docs might still be using older versions of NuGet (especially when their CI system defaults to some older version, or they have build scripts that use specific versions), I think it's worthwhile mentioning which version of nuget.exe this is supported in.

Reply:
Thanks @zivkan.
I've updated the decription and also noted that option will be available only from specific version. For now, I've set the version as `6.6`. Let's see when the PR gets merged.